src/types/message.ts

```diff
@@ -20,36 +20,42 @@ interface MessageBase {
 }
 
 // Definitions for all messages from GCS to vehicles.
-
-export interface StartMessage extends MessageBase {
-  type: 'start';
+export interface GeofenceMessage extends MessageBase {
```

Comment:
lets put the GeofencingMessage in the same order as you had it in the docs for consistency